### DISCUSSION TITLE
Power plans don't change when I switch performance modes in G-Helper

### DISCUSSION THREAD

**Discussion by Sageo69** [seerge/g-helper#1584] [Q&A]
Am I missing something or it's supossed to be like that? On all 3 modes system stays on Balanced power plan, even tho I've selected different "Windows Power Mode" setting for each of them. Does it change the settings of the power plan itself every time I'm switching the mode?

**Comment by seerge** [comment]
@Sageo69 hello, power plans are not supposed to change. Power modes do.

Please [check the readme](https://github.com/seerge/g-helper#rocket-performance-modes) :)
